cbmc_viewer/make_clause.py

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env python3
+
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+# -*- mode: python-mode -*-
+
+"""List CBMC solver query complexity metrics."""
+
+
+import argparse
+import sys
+
+from cbmc_viewer import clauset
+from cbmc_viewer import optionst
+
+def create_parser():
+    """Command line parser."""
+
+    parser = argparse.ArgumentParser(
+        description="List CBMC solver query complexity metrics."
+    )
+
+    optionst.clause(parser)
+    optionst.srcdir(parser)
+
+    parser.add_argument(
```

Comment:
These are arguments I'd add to optionst.py

Reply:
Done.